teachers_digital_platform/migrations/0010_load_initial_data.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from django.db import migrations, models
+
+from sys import path
+from django.core import serializers
+
+fixture_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), '../fixtures'))
+fixture_filename = 'tdp_initial_data.json'
+
+
+def load_fixture(apps, schema_editor):
+    fixture_file = os.path.join(fixture_dir, fixture_filename)
+
+    fixture = open(fixture_file, 'rb')
```

Comment:
MInor suggestion: use a context manager (`with open(fixture_file, 'rb') as fixture:`) here instead. This avoids the need for the `close` and properly closes the file should something go wrong. See [the docs](https://docs.python.org/2/tutorial/inputoutput.html#methods-of-file-objects) (look down for "good practice").